runtime/tests/interpreter/values_test.go

```diff
@@ -1390,182 +1413,135 @@ func randomArrayValue(inter *interpreter.Interpreter, currentDepth int) interpre
 	)
 }
 
-func randomCompositeValue(
```

Comment:
This was a variation of `generateRandomCompositeValue`. I merged the two functions into one